### PR TITLE
Samples redirect

### LINKED ISSUE #214: Add 2 more levels : Dye and Device

### PR DESCRIPTION
**Pull request by maasencioh**
## New menu full screen
image

## New menu responsive
image

Closes #214

### CODE REVIEW

File: front/src/components/Nav/Menu.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  routes: RouteType[];
+}
+
+export function Menu({ title, routes }: MenuProps) {
+  const { pathname } = useLocation();
+  const isMatch = routes.reduce(
+    (acc, route) =>
+      acc || minimatch(pathname, route.pathmatch || route.pathname),
+    false,
+  );
```
Comment (targos) [inline front/src/components/Nav/Menu.tsx]: I think we could use the new `useMatch` hook from react-router?

File: front/src/components/Nav/RouteLink.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import { RouteType } from './types';
+
+interface RouteLinkProps {
+  route: RouteType;
+}
+
+export default function RouteLink({ route }: RouteLinkProps) {
+  const { pathname } = useLocation();
+  const isMatch = minimatch(pathname, route.pathmatch || route.pathname);
```
Comment (targos) [inline front/src/components/Nav/RouteLink.tsx]: ditto

**Comment by targos** [comment]
I have nothing more to add.

**Comment by maasencioh** [comment]
Thanks @targos 👍

**Pull request merged**